patches/chromium/printing.patch

```diff
@@ -184,19 +165,17 @@ index 34d6001efd674bd49031845e8257c468d894258e..4f29c2ac96e50cdfe49669802edfd9ab
    // Remember the ID for `rfh`, to enable checking that the `RenderFrameHost`
    // is still valid after a possible inner message loop runs in
    // `DisconnectFromCurrentPrintJob()`.
-@@ -226,7 +247,10 @@ bool PrintViewManagerBase::PrintNow(content::RenderFrameHost* rfh) {
+@@ -216,7 +238,8 @@ bool PrintViewManagerBase::PrintNow(content::RenderFrameHost* rfh) {
  #endif
  
    SetPrintingRFH(rfh);
 -  CompletePrintNow(rfh);
-+  //CompletePrintNow(rfh);
```

Comment:
nit: `#if 0` would be better than deleting lines.

Reply:
restored and `#if 0`'ed out in 825b9b239d75adb914e7ff565e4cb097ac22d380